.vale/styles/Vocab/word_list/accept.txt

```diff
@@ -78,3 +78,27 @@ AMIs
 [Cc]ancelation
 Brotli
 zlib
+datetime
+GZip
+videogame
+toc
+gcp
+faq
+synon
+npm
+html
+Maneu
+Hoxie
+Bleve
+Yonik
+Seeley
+MBs
+url
+masterKey
```

Comment:
Doesn't this rule ignore text wrapped in `<code>` elements? I'm curious about this one, as well as a few others that normally would be capitalized (`toc`, `gcp`, `faq`, `npm`, `url`).

Reply:
gcp, faq, npm, url are all used in alt text. You can't use backticks in alt text, so I added them here to reduce the warnings.
toc isn't used in backticks. It's [toc]. We only have one page that uses it. We should remove it as well.

Reply:
Ok, that makes sense. However there's something going on here related to acronyms/case-sensitivity, no? I mean, we have `GCP` in an article title and it's not setting off any flags. So why is the lowercase causing problems? And shouldn't the lowercase `gcp` in the alt-text be uppercase, anyway (same for `faq` and `url` IMO)?

Reply:
- faq, npm, synon: we're using GIFs that search for these terms in lowercase
- gcp: only used once in the telemetry table in the "Example" column. We cannot use uppercase there
- url: "url" is a field in Postman. We added `"(.*?)"` so this won't be an issue anymore

Reply:
I see. Is there a way to make this rule ignore alt text?

Reply:
We could, but then Vale would ignore all alt text which isn't a good idea

Reply:
The reason I am pushing for this is that adding `url, `gcp`, and `faq` to our exceptions list will make our linter less effective since we never actually want to write them like this in body text.